### PR TITLE
Docs fixes about Slide's processed_path

### PR DESCRIPTION
**Pull request by alessiamarcolini**

### CODE REVIEW

File: src/histolab/slide.py [hunk 1]
```diff
@@ -210,12 +210,12 @@ def name(self) -> str:
 
     @lazyproperty
     def processed_path(self) -> str:
-        """Retrieve the path to store processed files generated from the slide.
+        """Retrieve the path to store the tiles generated from the slide.
```
Comment (ernestoarbitrio) [inline src/histolab/slide.py:213]: I think that considering that this is a lazy property the docstring shouldn't contain the word Retrieve cause it doesn't retrieve anything it's a property

**Review by ernestoarbitrio — CHANGES_REQUESTED** [review]
Per comments

File: src/histolab/slide.py [hunk 2]
```diff
@@ -200,7 +200,7 @@ def locate_mask(
 
     @lazyproperty
     def name(self) -> str:
-        """Retrieve the slide name without extension.
+        """Return the slide name without extension.
```
Comment (ernestoarbitrio) [inline src/histolab/slide.py:203]: A property doesn't compute any action it's a property and for this the docstring should be something without the verbs. I'd use Slide name without extension.

File: src/histolab/slide.py [hunk 3]
```diff
@@ -210,12 +210,12 @@ def name(self) -> str:
 
     @lazyproperty
     def processed_path(self) -> str:
-        """Retrieve the path to store processed files generated from the slide.
+        """Return the path to store the tiles generated from the slide.
```
Comment (ernestoarbitrio) [inline src/histolab/slide.py:213]: Same

**Review by ernestoarbitrio — CHANGES_REQUESTED** [review]
Last iteration sorry 🙂

**Comment by alessiamarcolini** [comment]
@ernestoarbitrio I updated all the docstrings of lazyproperties, requested review

**Review by ernestoarbitrio — APPROVED** [review]
LGTM :)

**Pull request merged**